CHANGELOG.md

```diff
@@ -22,10 +22,14 @@
    
   The `within expression` enables checking whether a feature is inside a pre-defined geometry set/boundary or not. This `within expression` returns a boolean value, `true` indicates that the feature being evaluated is inside the geometry set. The returned value can be then consumed as input by another expression or used directly by a paint/layer property.
  
-  Support for using `within expression` with layout propery will be implemented separately.
+  Support for using `within expression` with layout property will be implemented separately.
 
-- [core] Add support for using `within expression` with layout propery. ([#16194](https://github.com/mapbox/mapbox-gl-native/pull/16194))
+- [core] Add support for using `within expression` with layout property. ([#16194](https://github.com/mapbox/mapbox-gl-native/pull/16194))
 
+- [core] Add support for `in expression`. ([#16162](https://github.com/mapbox/mapbox-gl-native/pull/16162))
+   
+  The `in expression` enables checking whether a Number/String/Boolean type needle is in a String/Array and returns a boolean value.
```

Comment:
It might be confusing to use the term "needle" here, at least without also using the corresponding "haystack" to make the analogy clear. I'd also probably put them in quotation marks to make it clear that these are terms of art and not specific technical terms.